developers/weaviate/client-libraries/python.md

```diff
@@ -12,11 +12,8 @@ import UserTestCode from '!!raw-loader!./_includes/examples.py';
 import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
 
 :::caution Beta version
-The Python client is currently in beta, and we want to hear from you.
 
-You can test the new client locally, or on paid instances of Weaviate Cloud Services (WCS). It is not yet available on the free (sandbox) tier of WCS.
-
-If you notice any bugs, or have any feedback, please let us know on [this forum thread](https://forum.weaviate.io/t/python-v4-client-feedback-megathread/892)
+ The Python client is currently in beta, and we want to hear from you. You can test the new client locally, or on paid instances of Weaviate Cloud Services (WCS). It is not yet available on the free (sandbox) tier of WCS. If you notice any bugs, or have any feedback, please let us know on [this forum thread](https://forum.weaviate.io/t/python-v4-client-feedback-megathread/892)
```

Comment:
Whitespace changes - the admonition box doesn't leave blank lines so this layout is deceiving.